app/src/export.js

```diff
@@ -26,8 +29,11 @@ export default function Export() {
       const result = await API.get(apiName, path, myInit);
       const transcribe = result['data']['transcribe'];
       const comprehend = result['data']['comprehend'];
+      const soap = result['data']['soap'];
```

Comment:
NIT: can we name it as soap_notes?

Reply:
Should follow camelCase in JS, so `soapNotes`